arm-network/2016-09-01/swagger/network.json

```diff
@@ -5117,6 +5124,27 @@
       ],
       "description": "UrlPathMap of application gateway"
     },
+    "ApplicationGatewayWebApplicationFirewallConfiguration": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether web application firewall is enabled"
```

Comment:
does this have a default value? Will the following payload make sense: 
```
{
   firewallMode: "Prevention"
}
```

Should enabled be made required?